lib/cisco_node_utils/cmd_ref/vxlan_vtep.yaml

```diff
@@ -1,6 +1,6 @@
 # vxlan_vtep
 ---
-_exclude: [N3k, N5k, N6k, N7k, ios_xr]
```

Comment:
Since N7k is now an included platform, do you need to add a N7k entry to the `mt_lite_support` yaml entry?